community/modules/remote-desktop/chrome-remote-desktop/outputs.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "startup_script" {
+  description = "script to load and run all runners, as a string value."
+  value       = module.client_startup_script.startup_script
+}
+
+output "name" {
```

Comment:
Here, since you don't have to do any terraform processing of this output on the yaml anymore, you could rename this to instance_name and have wait just use it.

Reply:
Done.